kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -225,14 +336,38 @@
   /**
    * Update a {@link Dataset} for the given dataset or view URI.
    * <p>
-   * URI formats are defined by {@code Dataset} implementations, but must begin
-   * with "dataset:" or "view:".
+   * You can add columns, remove columns, or change the data type of columns 
+   * in your dataset, provided you don't attempt a change that would corrupt 
+   * the data. Kite follows the guidelines in the Avro schema. See 
```

Comment:
Changed paragraph in all 4 update instances to:

   * You can add columns, remove columns, or change the data type of columns 
   * in your dataset, provided you don't attempt a change that is incompatible
   * with written data. Avro defines rules for compatible schema evolution. See 
   * <a href="http://kitesdk.org/docs/current/Schema-Evolution.html">Schema
   * Evolution</a>.